From fc84fbc9bc84a6fc0d94e0ed7ab709e6b2791072 Mon Sep 17 00:00:00 2001
From: SkyBird233 <52884766+SkyBird233@users.noreply.github.com>
Date: Tue, 29 Sep 2026 22:14:03 +0800
Subject: [PATCH 1/2] Remove pnpm-managed cargo sources

We cannot use pnpm to install them because we are building pnpm.
---
 .cargo/config.toml | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/.cargo/config.toml b/.cargo/config.toml
index 0a86ab5f62..3ac6b6788a 100644
--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -2,18 +2,3 @@
 # Do not append `--` or it will break IDEs
 codecov = "llvm-cov nextest --workspace --exclude pnpm-integrated-benchmark --exclude pnpm-micro-benchmark --exclude pnpm-registry-mock --ignore-filename-regex tasks"
 
-# >>> pnpm-managed cargo sources >>>
-[source.crates-io]
-replace-with = "pnpm-crates-io"
-
-[source.pnpm-crates-io]
-directory = ".pnpm/crates/crates-io"
-
-[source.pnpm-git]
-directory = ".pnpm/crates/git"
-
-[source."git+https://github.com/pnpm/node-semver-rs?rev=3d74d6b7f4de1de2006cf2f7663aac09de65b1f6"]
-git = "https://github.com/pnpm/node-semver-rs"
-rev = "3d74d6b7f4de1de2006cf2f7663aac09de65b1f6"
-replace-with = "pnpm-git"
-# <<< pnpm-managed cargo sources <<<
-- 
2.56.0

